etc/profile-m-z/xonotic.profile

```diff
@@ -32,7 +32,6 @@ caps.drop all
 netfilter
 nodvd
 nogroups
-noinput
```

Comment:
Xonotic gamepad support only functions on Windows.

Reply:
I did test all the games I was able to compile/install on my Arch box and Xonotic does have a working gamepad support on Linux as well. Here's a screenshot from a session running on Arch:
![obrázok](https://user-images.githubusercontent.com/1406222/137804801-bc485f58-690f-455b-bc93-28719cfa9aba.png)